rosidl_typesupport_fastrtps_cpp/include/rosidl_typesupport_fastrtps_cpp/message_type_support.h

```diff
@@ -19,25 +19,51 @@
 
 #include <fastcdr/Cdr.h>
 
+/// Encapsulates the callbacks for getting properties of this rosidl type.
+/**
+ * These callbacks are implemented in the generated sources.
+ */
 typedef struct message_type_support_callbacks_t
 {
+  /// The C++ namespace of this message.
   const char * message_namespace_;
+
+  /// The typename of this message.
   const char * message_name_;
 
-  // Function for message serialization
+  /// Function for message serialization
```

Comment:
This and below are pointer functions, shouldn't this be reflected in the description? Either replacing with callback or using "Pointer to function"
```suggestion
  /// Callback for message serialization
```

Reply:
Fixed.